src/app/Http/Controllers/Admin/UniquePageCrudController.php

```diff
@@ -0,0 +1,245 @@
+    public function buttons($page)
+    {
+        $openButton = $page->getOpenButton();
+        $revisionsButton = view('crud::buttons.revisions', ['crud' => $this->crud, 'entry' => $page]);
+
+        return $openButton.' '.$revisionsButton;
+    }
+
+    /**
+     * Create the page by slug.
+     *
+     * @param $slug
+     * @return mixed
+     */
+    public function createMissingPage($slug)
+    {
+        $slugs = $this->getUniqueSlugs();
+
+        if (! $slugs->has($slug)) {
+            abort(404);
+        }
+
+        $page = $slugs->pull($slug);
+        $model = $this->crud->model;
+
+        return $model::create([
+            'template' => $page,
+            'name'     => camel_case($page),
+            'title'    => camel_case($page),
+            'slug'     => $slug,
+        ]);
+    }
+
+    /**
+     * Display the revisions for specified resource.
+     *
+     * @param $slug
+     * @param $id
+     * @return Response
+     */
+    public function uniqueRevisions($slug, $id)
+    {
+        $model = $this->crud->model;
+        $entry = $model::findBySlugOrFail($slug);
+
+        $this->uniqueSetup($entry);
+
+        return parent::listRevisions($entry->id);
+    }
+
+    /**
+     * Restore a specific revision for the specified resource.
+     *
+     * Used via AJAX in the revisions view
+     *
+     * @param string $slug
+     * @param int $id
+     *
+     * @return JSON Response containing the new revision that was created from the update
+     * @return HTTP 500 if the request did not contain the revision ID
+     */
+    public function restoreUniqueRevision($slug, $id)
+    {
+        $model = $this->crud->model;
+        $entry = $model::findBySlugOrFail($slug);
+
+        $this->uniqueSetup($entry);
+
+        return parent::restoreRevision($id);
+    }
+
+    /**
+     * Setup the controller for the entry.
+     *
+     * @param $entry
+     */
+    protected function uniqueSetup($entry)
+    {
+        $this->setRoute($entry->slug);
+
+        $this->addDefaultPageFields($entry);
+        $this->crud->setEntityNameStrings($this->crud->makeLabel($entry->template), $this->crud->makeLabel($entry->template));
+
+        $this->{$entry->template}();
+    }
+
+    /*
+    |--------------------------------------------------------------------------
+    | SaveActions overrides
+    |--------------------------------------------------------------------------
+    */
+
+    /**
+     * Overrides trait version to remove 'save_and_back' and 'save_and_new'.
+     *
+     * @return [type] [description]
+     */
+    public function getSaveAction()
+    {
+        $saveCurrent = [
+            'value' => $this->getSaveActionButtonName('save_and_back'),
+            'label' => $this->getSaveActionButtonName('save_and_back'),
+        ];
+
+        return [
+            'active'  => $saveCurrent,
+            'options' => [],
+        ];
+    }
+
+    /**
+     * Override trait version to not update the session variable.
```

Comment:
I forget why?

Reply:
we have no "save actions" here and overwrite with `save_and_back`.
This would geht default for this session in all Crud-Views.
As we don't want to overwrite behaviour when user can't choose, we need this (empty) implementation

Reply:
add some more comment to make this clear ;)